examples/connectivity/aws-connection/main.tf

```diff
@@ -31,14 +31,18 @@ resource "equinix_ecx_l2_connection" "example" {
   authorization_key = var.aws_account_id
 }
 
-resource "equinix_ecx_l2_connection_accepter" "example" {
-  connection_id = equinix_ecx_l2_connection.example.id
-  access_key    = var.aws_access_key
-  secret_key    = var.aws_secret_key
+resource "aws_dx_connection" "example" {
```

Comment:
'equinix_ecx_l2_connection' resource creates the connection in the AWS side on your behalf and therefore the 'aws_dx_connection' resource is not required

Reply:
If we don't need an dx_ec2_connection resource, how should we get the connection I'd that AWS needs in their APIs and Terraform resources?